jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -11,11 +11,17 @@
                                         FileImportSelectPluginComponent, HasFileImportSelect,
                                         with_spinner)
 
+from jdaviz.core.template_mixin import TableMixin
+from jdaviz.core.user_api import PluginUserApi
+
+
+
 __all__ = ['Catalogs']
 
 
 @tray_registry('imviz-catalogs', label="Catalog Search")
-class Catalogs(PluginTemplateMixin, ViewerSelectMixin, HasFileImportSelect):
+
```

Comment:
not sure if this fails codestyle or not, but we usually don't have an empty line after decorators
```suggestion
```